daxa/config.py

```diff
@@ -70,3 +70,7 @@
 # Just the names of the errors in two handy constants
 SASERROR_LIST = errors["ErrName"].values
 SASWARNING_LIST = warnings["WarnName"].values
+
+# Reading in the data available in the Cal-PV release
+CalPV_info = pd.read_csv(pkg_resources.resource_filename(__name__, "files/eROSITACalPV_info.csv"), header="infer")
```

Comment:
CalPV_info should be all uppercase, that is the Python PEP convention for constants, it distinguishes them from normal variables (though not in their behaviour)